quoridor-logic-server/src/main/java/com/rl/mpquoridor/models/actions/TurnAction.java

```diff
@@ -0,0 +1,4 @@
+package com.rl.mpquoridor.models.actions;
+
+public abstract class TurnAction {
```

Comment:
why not interface? it not that problematic but if its not contains common logic for the actions it could be interface